e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -514,10 +514,23 @@ test.describe('Notebook entry tests', () => {
     const childItem = page.locator('li:has-text("List Item 2") ol li:has-text("Order 2")');
     await expect(childItem).toBeVisible();
 
-    // Blocks
-    const blockTest = '```javascript\nconst foo = "bar";\nconst bar = "foo";\n```';
-    await nbUtils.enterTextEntry(page, blockTest);
+    // Code Blocks
```

Comment:
Should we have a visual test for this feature as well?

Reply:
We've got a ticket for visual markdown tests [here](https://github.com/nasa/openmct/issues/7108).